sources/VISMA/.go_import/runner.rb

```diff
@@ -60,19 +60,19 @@ def convert_source
     end
     puts "Processed #{imported_person_count} Persons."
 
-    # Notes must be owned by a coworker and the be added to
-    # organizations and notes and might refernce a person
-    puts "Trying to process Notes..."
-    imported_note_count = 0
+    # History must be owned by a coworker and the be added to
+    # organizations and histories and might refernce a person
```

Comment:
History must be owned by a coworker and then should reference
organization or deal and might reference a person